website/source/guides/advanced-scheduling/affinity.html.md

```diff
@@ -0,0 +1,231 @@
+
+```shell
+data_dir = "/opt/nomad/data"
+bind_addr = "0.0.0.0"
+datacenter = "dc2"
+
+# Enable the client
+client {
+  enabled = true
+...
+```
+After making the change on your chosen client node, restart the Nomad service
+
+```shell
+$ sudo systemctl restart nomad
+```
+
+If everything worked correctly, you should be able to run the `nomad` [node status][node-status] command and see that one of your nodes is now in datacenter `dc2`.
+
+```shell
+$ nomad node status
+ID        DC   Name              Class   Drain  Eligibility  Status
+3592943e  dc1  ip-172-31-27-159  <none>  false  eligible     ready
+3dea0188  dc1  ip-172-31-16-175  <none>  false  eligible     ready
+6b6e9518  dc2  ip-172-31-27-25   <none>  false  eligible     ready
+```
+
+### Step 2: Create a Job with the `affinity` Stanza
+
+Create a file with the name `redis.nomad` and place the following content in it:
+
+```hcl
+job "redis" {
+ datacenters = ["dc1", "dc2"]
+ type = "service"
+
+ affinity {
+   attribute = "${node.datacenter}"
+   value = "dc2"
+   weight = 100
+ }
+
+ group "cache1" {
+   count = 4
+
+   task "redis" {
+     driver = "docker"
+
+     config {
+       image = "redis:latest"
+       port_map {
+         db = 6379
+       }
+     }
+
+     resources {
+       network {
+         port "db" {}
+       }
+     }
+
+     service {
+       name = "redis-cache"
+       port = "db"
+       check {
+         name     = "alive"
+         type     = "tcp"
+         interval = "10s"
+         timeout  = "2s"
+       }
+     }
+   }
+ }
+}
+```
+Note that we used the `affinity` stanza and specified `dc2` as the
+value for the [attribute][attributes] `${node.datacenter}`. We used the value
+`100` for the [weight][weight] which will cause the Nomad schedular to rank nodes
+in datacenter `dc2` with a higher score. Keep in mind that weights can range
+from -100 to 100, inclusive. Negative weights serve as anti-affinities.
+
+### Step 3: Register the Job `redis.nomad`
+
+Run the Nomad job with the following command:
+
+```shell
+$ nomad run redis.nomad 
+==> Monitoring evaluation "11388ef2"
+    Evaluation triggered by job "redis"
+    Allocation "0dfcf0ba" created: node "6b6e9518", group "cache1"
+    Allocation "89a9aae9" created: node "3592943e", group "cache1"
+    Allocation "9a00f742" created: node "6b6e9518", group "cache1"
+    Allocation "fc0f21bc" created: node "3dea0188", group "cache1"
+    Evaluation status changed: "pending" -> "complete"
+==> Evaluation "11388ef2" finished with status "complete"
+```
+
+Note that two of the allocations have been placed on node `6b6e9518`. This is
+the node we configured to be in datacenter `dc2`. The Nomad scheduler has leaned
+towards placing the majority of the workload on this node because of the
+affinity we specified. All of the workload has not been placed on this node
```

Comment:
Wording suggestion "However, all the allocations have not been placed on that node because the scheduler considers other factors in the scoring such as bin packing, and avoiding placing too many instances of the same job on a node to prevent reduced capacity during a node level failure. We will look at scoring factors in the next few steps."